src/enemy.ts

```diff
@@ -5,8 +5,30 @@ import { DirectionsType } from './projectile';
 export class Enemy extends Tank {
   private _directions: DirectionsType[] = ['up', 'down', 'left', 'right'];
 
+  protected _upPos: string[][] = [
+    [' ', 'M', ' '],
+    ['@', 'M', '@'],
+    ['@', 'M', '@'],
+  ];
+  protected _downPos: string[][] = [
```

Comment:
Why this fiedls are protected instead of private?

Reply:
protected allows to use this field in inherited classes